remote/remote_state.go

```diff
@@ -26,6 +26,8 @@ type RemoteStateInitializer interface {
 
 	// Initialize the remote state
 	Initialize(config map[string]interface{}, terragruntOptions *options.TerragruntOptions) error
+
+	GetTerraformInitArgs(config map[string]interface{}) map[string]interface{}
```

Comment:
Please add a comment explaining what this method does and why it exists

Reply:
Done